docs/Introduction/Delay and Force.md

```diff
@@ -25,12 +25,12 @@ A delayed term evaluates when it is _forced_ using the `pforce` function. Forcin
 pforce :: Term s (PDelayed a) -> Term s a
 ```
 
-Thus, if we wanted a lazy `pif`, we could do the following:
+Thus, if we wanted a lazy `hif`, we could do the following:
```

Comment:
I think this _should_ be pif.
It's the part below that's incorrect.

Reply:
wdym part below, my edit or the function following?
```hs
-- | Utilizing Haskell level functions with `pdelay` and `pforce` to have lazy wrapper around `pif`.
hif :: Term s PBool -> Term s a -> Term s a -> Term s a
hif cond whenTrue whenFalse = pforce $ pif' # cond # pdelay whenTrue # pdelay whenFalse
```
If the former, I think it's hif because it's haskell-level, not plutarch level. But it would also be better to specify in the definition of hif at the top that it applies only to `Term`s, as that hints towards it being non-lazy.

Reply:
This should be called `pif'`, sorry for the very late response. It's not a Haskell-if after all.